src/aap_eda/api/serializers/activation.py

```diff
@@ -152,6 +152,9 @@ def _update_extra_vars_from_eda_credentials(
         if creating and any(key in user_inputs for key in secret_fields):
             vault_data.password_used = True
 
+        if not injectors or "extra_vars" not in injectors:
+            continue
```

Comment:
Is this `continue` correct? Those injectors without `extra_var` but have `env` or `file` inside will be skipped

Reply:
we want to purposely skip `env` and `file`, as this function is to extract extra vars and save them to `activation.extra_var` field. There are no fields for env and file in activation.